app/Main.hs

```diff
@@ -1,26 +1,46 @@
 module Main where
 
 import           Ampersand
+import           Control.Monad
 import           Data.List
 import qualified Data.List.NonEmpty as NEL (toList)
 
 main :: IO ()
 main =
  do opts <- getOptions
-    if showVersion opts || showHelp opts --HJO 20161127 TODO: There are more commands that do not 
-                                         --  need a single filename to be specified, 
-                                         --  like --sampleConfigFile. Currently, this
-                                         --  does not work properly. A more generic
-                                         --  approach for handling those options should
-                                         --  be thought of. 
-    then mapM_ putStr (helpNVersionTexts ampersandVersionStr opts)
-    else do { verboseLn opts $ ampersandVersionStr
+    verboseLn opts ampersandVersionStr
```

Comment:
Dit zorgt er voor, dat de versie mogelijk twee keer wordt afgedrukt, namelijk als --version --verbose wordt gevraagd. Dat is vast niet de bedoeling.
```suggestion
    when (verboseP opts || showVersion opts) (putStrLn ampersandVersionStr)
```
En dan moet je bij --version verder niets meer tonen.